Remote: Appsody java-microprofile project missing .appsody-config.yaml error

<!-- Please fill out the following form to report a bug. If some fields do not apply to your situation, feel free to skip them.-->

**Codewind version:**
**OS:**


**Che version:**
**IDE extension version:**
**IDE version:**
**Kubernetes cluster:**


**Description:**

Create an appsody `java-microprofile` project, I see this in `Turbine.log`:

```
[25/10/19 17:42:59 Turbine] [ERROR] [File Name: /file-watcher/server/dist/utils/logger.js | Function Name: logProjectError | Line Number: 202] e2676f10-f74e-11e9-b7ce-ddbd5f49a2b9: ShellExtensionProject.setLanguage : 70 | grep: .appsody-config.yaml: No such file or directory
```

Notes:
- I do not see this error with `nodes-express` project
- The file is clearly there in `/mounted-workspace/<projectname>/.appsody-config.yaml`
- When debugging through the code, things are slowed down such that this error is not encountered

Possibly it's a timing issue of the project files being uploaded, the `.appsody-config.yaml` file may not be copied to the right location when the above runs.

**Steps to reproduce:**


**Workaround:** 
<!-- Did you find a way to work around the bug? If so, please describe how you worked around it.-->

--- a/src/pfe/portal/routes/projects/remoteBind.route.js
+++ b/src/pfe/portal/routes/projects/remoteBind.route.js
@@ -243,7 +243,7 @@ async function syncToBuildContainer(project, filesToDelete, pathToTempProj, modi
     // We now need to remove any files that have been deleted from the global workspace
     await Promise.all(filesToDelete.map(oldFile => exec(`rm -rf ${path.join(globalProjectPath, oldFile)}`)));
     // now move temp project to real project
-    cwUtils.copyProject(pathToTempProj, project.workspace);
+    await cwUtils.copyProject(pathToTempProj, project.workspace);
     let projectRoot = getProjectSourceRoot(project);
     // need to delete from the build container as well
     if (!global.codewind.RUNNING_IN_K8S && project.projectType != 'docker' &&
@@ -362,7 +362,7 @@ async function bindEnd(req, res) {
 
     const pathToCopy = path.join(global.codewind.CODEWIND_WORKSPACE, global.codewind.CODEWIND_TEMP_WORKSPACE, project.name);
     // now move temp project to real project
-    cwUtils.copyProject(pathToCopy, project.workspace);
+    await cwUtils.copyProject(pathToCopy, project.workspace);
 
     let updatedProject = {
       projectID,

